v-next/hardhat/src/internal/cli/telemetry/analytics/analytics-subprocess.ts

```diff
@@ -0,0 +1,17 @@
+import { postJsonRequest } from "@ignored/hardhat-vnext-utils/request";
+
+// These keys are expected to be public
+const ANALYTICS_URL = "https://www.google-analytics.com/mp/collect";
```

Comment:
Are these the test or the production values?

Reply:
Production values

Reply:
I'll update with test variables and create a work item to remember to replace them before v3 is released